pages/login.css

```diff
@@ -0,0 +1,89 @@
+:root {
+    --text-color: #282828;
+}
+
+body {
+    width: 100%;
+    height: 100vh;
+    color: var(--text-color);
+    font-family: sans-serif;
+}
+
+.container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-height: 100vh;
+}
+
+.login-form {
+    width: 25rem;
+    height: 25rem;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    box-shadow: rgb(0 0 0 / 16%) 0px 1px 4px;
+    border-radius: 5px;
+}
+.heading {
+    margin-bottom: 2rem;
+    padding: 2rem 2rem 0 2rem;
+}
+.text-center {
+    text-align: center;
+}
+.input {
+    color: #333;
```

Comment:
You can make use of a variable here. Create one for it in the root just like you have done for text color